src/openforms/js/components/form/select.js

```diff
@@ -5,14 +5,37 @@ import {localiseSchema} from './i18n';
 
 const Select = Formio.Components.components.select;
 
-const values = [
+const optionsChoices = [
```

Comment:
If I see this correctly, then it's exactly the same as `OPTIONS_CHOICES` except for the `key` being `values` vs. `data.values`, no?

Reply:
Yes it's true, now that I moved the other parts to the `openForms` attribute, the rest is the same except the `values` component. I will refactor it!